SQL/0000-00-00-schema.sql

```diff
@@ -711,7 +711,7 @@ CREATE TABLE `mri_upload` (
   `UploadLocation` varchar(255) NOT NULL DEFAULT '',
   `DecompressedLocation` varchar(255) NOT NULL DEFAULT '',
   `InsertionComplete` tinyint(1) NOT NULL DEFAULT '0',
-  `Inserting` tinyint(1) NOT NULL DEFAULT '0',
+  `Inserting` tinyint(1) DEFAULT NULL,
```

Comment:
@MounaSafiHarab What does the "Inserting" column signify? If it's whether or not it's currently being inserted (which is what I would assume) it seems like not null default false is the right default..

Reply:
we do distinguish between NULL (for a new upload), inserting=1 meaning the pipeline is in progress, and inserting=0, meaning the insertion completed.
This distinction is needed for the 'Progress' column in Imaging Uploader.

Using InsertionComplete in conjunction with Inserting could potentially be used, but is not...